### PR TITLE
Update README file with new projects integration details

### PR DESCRIPTION
**Pull request by sradco**
This patch adds details on how other projects can integrate
and add their code to the logging role.

Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

**Comment by sradco** [comment]
Hi,
I will appreciate your review for this patch.
@nhosoi @nkinder @pcahyna @tabowling

File: README.md [hunk 1]
```diff
@@ -8,9 +8,30 @@ The `logging` role enables a RHEL admin/developer to deploy logging collectors o
...
+=================
+
+<!--ts-->
+   * [linux-system-roles Logging](#linux-system-roles-logging)
+   * [Table of contents](#table-of-contents)
+   * [Definitions](#definitions)
+   * [Deploy Default Logging Configuration Files](#deploy-default-logging-configuration-files)
+   * [Deploy Configuration Files](#deploy-configuration-files)
+      * [Inventory File](#inventory-file)
+      * [vars.yaml](#vars)
```
Comment (nhosoi) [inline README.md]: (#varsyaml)

File: README.md [hunk 2]
```diff
@@ -8,9 +8,30 @@ The `logging` role enables a RHEL admin/developer to deploy logging collectors o
...
+      * [Variables in vars.yaml](#variables-in-vars.yaml)
```
Comment (nhosoi) [inline README.md]: (#variables-in-varsyaml)

File: README.md [hunk 3]
```diff
@@ -8,9 +8,30 @@ The `logging` role enables a RHEL admin/developer to deploy logging collectors o
...
+      * [playbook.yaml](#playbook)
```
Comment (nhosoi) [inline README.md]: (#playbookyaml)

**Comment by nhosoi** [comment]
Looks good to me with a few link fix requests in the Table of Contents.

**Pull request closed without merging**